peers/app_request_netrwork_metrics.go

```diff
@@ -0,0 +1,46 @@
+package peers
+
+import (
+	"errors"
+	"github.com/prometheus/client_golang/prometheus"
+)
```

Comment:
```suggestion
import (
	"errors"
	
	"github.com/prometheus/client_golang/prometheus"
)
```
I believe we have to add a line separation stdlib from third party imports for linting to pass: https://stackoverflow.com/questions/59960394/file-is-not-goimports-ed-with-local-somepath

Reply:
Thank you for the review.
I have rebased due to conflicts with the upstream code.
I have also added latency measurement code for the infoAPI in the constructor.